src/index.js

```diff
@@ -1,11 +1,12 @@
 import routerStateReducer from './routerStateReducer';
-import reduxRouteComponent from './reduxRouteComponent';
+import reduxRouteComponent, {routerMiddleware} from './reduxRouteComponent';
```

Comment:
Totally makes sense, sorry i had to install linux for a number of reasons and now i'm getting started to work on a separate project, i'll be back to this in no time